config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml

```diff
@@ -4,19 +4,32 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: (devel)
-    api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/139"
+    controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
I see that the latest release is v0.2.5, which was released 23 days ago.  Is there any reason not to use the latest release?

Reply:
There's not - This was the automatically generated version. I've updated this to be 0.2.5 in all the CRDs.

Reply:
Did you manually update them to 0.2.5?  I think it is the version of controller-gen you installed on your system.  Can you update controller-gen on your system to be 0.2.5 and have this generated automatically?

Reply:
Can you answer my question?

Reply:
I did manually update it - to my knowledge, I haven't built controller-gen on my system. I can see it in my gopath modules, though, and am attempting to update it before re-creating the CRDs.

Reply:
Do you have kubebuilder installed?

Reply:
I haven't installed or used it before.

Reply:
What tool are you using to generate the CRDs?

Reply:
I'd been using `hack/update-crd`. Checking this now I see:
>  go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.4;
Looks the repo defaults to v0.2.4 if controller-gen isn't detected. Let me update this and push again this evening.

Reply:
That makes sense.  Thanks.